### PR TITLE
description component

### PR DESCRIPTION
**Pull request by shawn320**
# Description 📝

Create component that displays discription on the main page

## Taiga Link 🔗

https://taiga.dts-stn.com/project/pond-dts-experimental-benefit-finder/task/104?kanban-status=117

## Checklist ✅
- [x] created story for all visual components
- [x] created or modified necessary unit tests
- [ ] created or modified e2e tests
- [ ] modified README or any applicable documentation on the changes you made

### CODE REVIEW

File: src/components/atoms/PageDescription.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export function PageDescription(props) {
```
Comment (Moro-Code) [inline src/components/atoms/PageDescription.js:7]: Make sure you add a comment for the component description too

File: src/components/atoms/PageDescription.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  return <div>{props.children}</div>;
```
Comment (Moro-Code) [inline src/components/atoms/PageDescription.js]: should this be a div? It should be a p element so that it's semantic

**Review by Moro-Code — CHANGES_REQUESTED** [review]
Well done man! Minor changes needed. Also all commits must be verified. Please set up a GPG key or use SSH to push to the repo

File: src/components/atoms/PageDescription.js (hunk 1 above)
Comment (shawn320) [reply src/components/atoms/PageDescription.js:7]: Thanks, I'll make sure to document my work in the future 👍

**Review by Moro-Code — APPROVED** [review]
LGTM! Thanks for the work

**Pull request merged**